src/libraries/System.Runtime.Numerics/tests/System.Runtime.Numerics.Tests.csproj

```diff
@@ -58,6 +58,7 @@
     <Compile Include="$(CommonTestPath)System\GenericMathHelpers.cs" Link="Common\System\GenericMathHelpers.cs" />
     <Compile Include="BigIntegerTests.GenericMath.cs" />
     <Compile Include="ComplexTests.GenericMath.cs" />
```

Comment:
@tannergooding Jeff mentioned you have a method for automatically generating Generic Math tests. Can you walk me through that process at some point?